modules/cluster-logging-loki-tech-preview.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//cluster-logging-release-notes.adoc
+
+:_content-type: PROCEDURE
+[id="cluster-logging-loki-tech-preview_{context}"]
+:FeatureName: Loki Operator
+include::snippets/technology-preview.adoc[]
+
+[id="cluster-logging-about-loki"]
+= About the Loki Operator
+[role="_abstract"]
+Loki is currently offered as an alternative to Elasticsearch as a log store for the {logging}.
+
+= Deploying the Loki Collector
+
+.Deploy loki-operator under openshift-operators namespace
+[source, terminal]
+----
+oc create -f  aosqe-tools/logging/log_template/loki-operator/loki-og-sub.yaml
```

Comment:
It is better to guide user to deploy loki-operator using Webconsole.
To deploy Loki Operator via client.

- Deploy openshift-operators-redhat namespace if it doesn't exist
cat <<EOF | oc create -f -
kind: Namespace
apiVersion: v1
metadata:
  name: openshift-operators-redhat
  annotations:
    openshift.io/node-selector: ""
  labels:
    openshift.io/cluster-logging: "true"
    openshift.io/cluster-monitoring: "true"
EOF

- Deploy OperatorGroup under openshift-operators-redhat if no OperatorGroup exists

cat <<EOF |oc create -f -
apiVersion: operators.coreos.com/v1
kind: OperatorGroup
metadata:
  name: openshift-operators-redhat
  namespace: openshift-operators-redhat
spec: {}
EOF

- subscript to CatalogSource redhat-operators  to deploy loki-operator

cat <<EOF |oc create -f -
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: loki-operator
  namespace: openshift-operators-redhat
spec:
  channel: "candidate"
  installPlanApproval: Automatic
  name: loki-operator
  source: redhat-operators
  sourceNamespace: openshift-marketplace
EOF